src/main/util.ts

```diff
@@ -136,10 +137,10 @@ const getMetadataForVideo = (video: string) => {
 /**
  * Writes video metadata asynchronously and returns a Promise
  */
- const writeMetadataFile = async (videoPath: string, metadata: any) => {
+ const writeMetadataFile = async (videoPath: string, metadata: Metadata) => {
```

Comment:
This needs to be `any` because this method is also called when you toggle "Protected" on a video and at that point the structure is NOT `Metadata`. Alternatively, and probably better, `Metadata` could be refactored to an interface `IVideoMetadata` which would then be OK since it just needs a compatible structure to pass the type check.

Reply:
Ah I think this bit me and caused lots of confusion. Thanks for pointing it out.

Reply:
I'd like to improve this with the interface as part of this so keeping open